### ISSUE TITLE
Error downloading on SLES 15 SP4 (and others)

### ISSUE THREAD

**Issue by pat-s** [rstudio/r-builds#135]
Tested a few others, seems like this permission issue applies to various DL URLs.

```bash
Enter version to install: (<ENTER> for latest)
4.2.1
Downloading installer from https://cdn.rstudio.com/r/opensuse-154/pkgs/R-4.2.1-1-1.x86_64.rpm...
Downloading https://cdn.rstudio.com/r/opensuse-154/pkgs/R-4.2.1-1-1.x86_64.rpm...
--2022-08-23 12:33:51--  https://cdn.rstudio.com/r/opensuse-154/pkgs/R-4.2.1-1-1.x86_64.rpm
Resolving cdn.rstudio.com (cdn.rstudio.com)... 108.138.17.43, 108.138.17.36, 108.138.17.62, ...
Connecting to cdn.rstudio.com (cdn.rstudio.com)|108.138.17.43|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2022-08-23 12:33:51 ERROR 403: Forbidden.


We were unable to download the installation bundle.
```

**Comment by glin** [comment]
Sorry about that, it's taking a bit longer to get SLES 15 SP4 and RHEL 9 binaries built. Support was added recently (https://github.com/rstudio/r-builds/pull/134), but the build images for SUSE 15.4 unexpectedly started failing as soon as I merged that PR. I'm working on a fix for it now, and will comment again when that's fixed.

So for now, RHEL 9 and SLES 15 SP4 builds are missing, and the CDN URL reports 403 although it should really be more like a 404 error.
